### ISSUE TITLE
Wow, I didn't know they allowed scams on GitHub...

### ISSUE THREAD

**Issue by ggtylerr** [pcsx4dev/orion-engine#1]
Well, technically it's a fork, but still. That's pretty fucking scummy, don't you think?

**Comment by Halamix2** [comment]
Nope, not even a proper fork, forks add info about upstream repo under the name, it's just copy-pasted files and then made into fresh git repo with no history

**Comment by RobLoach** [comment]
Under no circumstances will Users upload, post, host, or transmit any Content to any repositories that:
- is libelous, defamatory, or fraudulent
- contains or installs any active malware or exploits, or uses our platform for exploit delivery (such as part of a command and control system)

Report this user. You give emulation a bad name and rep.

**Comment by davwheat** [comment]
"report this user"

No public users for the organisation but... https://github.com/pcsx4emu

This is *one* of the accounts on the org.
